monitor_other_boards.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python3
+from datetime import datetime
+import subprocess
+
+import slack_reporter
+
+#                SSH target               name
+other_boards = (("odroid@odroid",         "odroid-C4"),
+                ("orangepi@oliviaorange", "orange pi zero 2"),
+                ("viam@viam",             "Up 4000"),
```

Comment:
Discussion question: I'm slightly uncomfortable having this list merged into the repo, but even more uncomfortable trying to update it on the monitor without having it in a central location. Is there a different way to do this? Is there a security issue with making this list public?